src/MSBuild/XMake.cs

```diff
@@ -1749,6 +1749,16 @@ private static List<BuildManager.DeferredBuildMessage> GetMessagesToLogInBuildLo
                     MessageImportance.Low),
             };
 
+            if (NativeMethodsShared.IsWindows)
+            {
+                messages.Add(
+                    new BuildManager.DeferredBuildMessage(
+                        ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword(
+                        "LongPathsEnabled",
+                        NativeMethodsShared.TryIsLongPathsEnabledRegistry() ? 1 : 0),
```

Comment:
I think I'd rather see "true"/"false" in the logs

```suggestion
                        NativeMethodsShared.TryIsLongPathsEnabledRegistry()),
```
```suggestion
                        NativeMethodsShared.TryIsLongPathsEnabledRegistry() ? 1 : 0),
```